### PR TITLE
Added assignment of week 2

### PR DESCRIPTION
**Pull request by Dibya12345**
This is the assignment of Week 2

### CODE REVIEW

**Comment by subhadip-pesto** [comment]
Please fix all places where you have maintained the alignment with position:relative and top/right/left/botto, properties.

File: Week-2/Question(2.2)/style.css [hunk 1]
```diff
@@ -0,0 +1,140 @@
...
+.picture{
+    width: 705px;
+    height: 664px;
+    background-color: #fff;
+    /* border: 2px solid black; */
+}
+.picture h3{
+    text-align: center;
+    font-size: 34px;
+    position: relative;
```
Comment (subhadip-pesto) [inline Week-2/Question(2.2)/style.css]: Please fix all places where you have maintained the alignment with position:relative and top/right/left/bottom, properties.

**Comment by Dibya12345** [comment]
Sure Do I have to remove all the position property

**Review by subhadip-pesto — CHANGES_REQUESTED** [review]

File: Week-2/Question(2.2)/style.css [hunk 2]
```diff
@@ -0,0 +1,140 @@
...
+    top: 13px
+}
+.picture img{
+    width: 199px;
+    height: 200px;
+    position: relative;
```
Comment (subhadip-pesto) [inline Week-2/Question(2.2)/style.css]: here also

**Review by subhadip-pesto — CHANGES_REQUESTED** [review]

File: Week-2/Question(2.2)/style.css (hunk 2 above)
Comment (Dibya12345) [reply Week-2/Question(2.2)/style.css]: I resolved it

File: Week-2/Question(2.2)/style.css (hunk 1 above)
Comment (Dibya12345) [reply Week-2/Question(2.2)/style.css]: Resolved it can u check now

**Review by subhadip-pesto — APPROVED** [review]

**Pull request merged**